src/cljs/athens/views/athena.cljs

```diff
@@ -1,5 +1,6 @@
 (ns athens.views.athena
   (:require
+    ["@material-ui/core/Dialog" :default Dialog]
```

Comment:
That's a bit too early.
We have not made the decision if we'll go with MUI or TailwindCSS, we have to test both with Steward.
https://github.com/athensresearch/athens/discussions/1316#discussioncomment-862581

Reply:
It's "Stuart" btw :)